client/components/Scenario/index.jsx

```diff
@@ -149,13 +151,14 @@ class Scenario extends Component {
             const isLastSlide = index === contents.length - 1;
             slides.push(
                 <ContentSlide
+                    isLastSlide={isLastSlide}
                     key={index}
                     slide={slide}
-                    isLastSlide={isLastSlide}
                     onClickBack={this.getOnClickHandler('back')}
                     onClickNext={this.getOnClickHandler(
                         isLastSlide ? 'finish' : 'next'
                     )}
+                    onResponseChange={onResponseChange}
```

Comment:
This is essentially the same fix in @evmiguel's audio response PR. I will remove this once that code lands and I rebase.